src/_support/Page/Acceptance/Administrator/UserNotesListPage.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * @package     Joomla.Test
+ * @subpackage  AcceptanceTester.Page
+ *
+ * @copyright   Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+namespace Page\Acceptance\Administrator;
+
+/**
+ * Acceptance Page object class to user list page.
+ *
+ * @category  Users
+ * @package   Page\Acceptance\Administrator
+ * @copyright 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license   GNU General Public License version 2 or later; see LICENSE.txt
+ * @since     4.0.0
+ */
+class UserNotesListPage
+{
+	// Include url of current page
+	public static $url = '/administrator/index.php?option=com_users&view=notes';
+
+	/**
+	 * Option one in list
+	 *
+	 * @var    array
+	 * @since  4.0.0
+	 */
+	public static $optionOne = '#cb0';
```

Comment:
not exactkly what i had in mind, when i asked you to rename to something more explicit. 
What do you think about `checkboxOfFirstListItem`?